src/Components/PresentationalComponents/Header/Breadcrumb.js

```diff
@@ -1,24 +1,26 @@
 import React from 'react';
 import propTypes from 'prop-types';
-import { Link } from 'react-router-dom';
+import { InsightsLink } from '@redhat-cloud-services/frontend-components/InsightsLink';
 import { Breadcrumb as PfBreadcrumb, BreadcrumbItem } from '@patternfly/react-core';
 import { Skeleton } from '@redhat-cloud-services/frontend-components/Skeleton';
 
-const Breadcrumb = ({ breadcrumbs }) => (
-    <PfBreadcrumb key="PfBreadcrumb" className="pf-u-mb-md">
-        {breadcrumbs?.map(item => (
-            <BreadcrumbItem key={item.title} isActive={item.isActive}>
-                {
-                    item.isLoaded
-                        ? item.isActive
-                            ? item.title
-                            : <Link to={item.to}>{item.title}</Link>
-                        : <Skeleton className="breadcrumb-skeleton" />
-                }
-            </BreadcrumbItem>
-        ))}
-    </PfBreadcrumb>
-);
+const Breadcrumb = ({ breadcrumbs }) => {
+    return (
+        <PfBreadcrumb key="PfBreadcrumb" className="pf-u-mb-md">
+            {breadcrumbs?.map(item => (
+                <BreadcrumbItem key={item.title} isActive={item.isActive}>
+                    {
+                        item.isLoaded
+                            ? item.isActive
+                                ? item.title
+                                : <InsightsLink to={`/${item.to}`}>{item.title}</InsightsLink>
```

Comment:
If you are using `InsightsLink` here the path/`to` can probably stay absolute with a leading `/`

Reply:
@bastilian I don't know what you mean here.